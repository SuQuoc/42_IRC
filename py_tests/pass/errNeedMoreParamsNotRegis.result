:AfterLife 464  :Password incorrect
