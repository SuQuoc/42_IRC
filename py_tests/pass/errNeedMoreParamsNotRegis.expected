:AfterLife 461 * PASS :Not enough parameters
